catalog/ui/catalog-ui-search/src/main/webapp/component/search-form-selector/search-forms-selector.view.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
We should update the folder name and the component name to match (one is plural for forms and the other isn't).

In addition, we should update the tag name used to be the same as the folder name (`search-form-selector`).

Reply:
renamed